### ISSUE TITLE
rails 3.1 compatibility

### ISSUE THREAD

**Issue by wspider** [binarylogic/searchlogic#155]
Please help me in searchlogic for rails 3.1.1. my whole application using searchlogic. I dont know what i do now its not working with Rails 3.1.1. can you people help us its work for rails 3.1.1 and above...

**Comment by andreydjason** [comment]
Hi, as now, maybe you can use this fork: https://github.com/railsdog/searchlogic it (should) works for Rails 3.x

**Comment by johndouthat** [comment]
It only works with 3.0.x :/

**Comment by andreydjason** [comment]
Thats right, sorry I miss it, besides, I'll need to find some another gem or kind of to do the "search thing"...

**Comment by cuzic4n** [comment]
bump

**Comment by binarylogic** [comment]
This is coming

**Comment by Makimeishu** [comment]
May I know whether it supports rails 3.2?

**Comment by tilsammans** [comment]
I suggesst everyone bite the bullet and rewrite using Ransack.
